package/boot/uboot-rockchip/Makefile

```diff
@@ -229,6 +229,13 @@ define U-Boot/nanopi-r6s-rk3588s
     friendlyarm_nanopi-r6s
 endef
 
+define U-Boot/nanopc-t6-rk3588
```

Comment:
this should put above nanopi-r6s-rk3588s

Reply:
fixed